src/registration/utils.py

```diff
@@ -20,6 +18,12 @@ def get_site(request):
     not installed.
 
     """
+    from django.contrib.sites.models import Site
+    try:
+        from django.contrib.sites.models import RequestSite
+    except ImportError:
+        from django.contrib.sites.requests import RequestSite
```

Comment:
We can't locate it in `compat.py`. Because of `AppRegistoryNotReady` exception.

In Django 1.9, We need to load models on runtime. So we can't load models on initialise.